README.md

```diff
@@ -2,3 +2,338 @@
 
 This is a DRAFT standard for building APIs in Hackney council - it is subject
 to review and change.
+
+Many of the standards set out in this document are based on the
+[White House Web API Standards](https://github.com/WhiteHouse/api-standards),
+with some specifics from the
+[Google JSON StyleGuide](https://google.github.io/styleguide/jsoncstyleguide.xml#Date_Property_Values)
+
+## Data Properties
+
+- Property names should be in lower camel case
+  - Good: `"contactTelephoneNumber"`
+  - Good: `"priority"`
+
+- Property names should use the language that Hackney uses rather than language used by any back end system:
+  - Good: `"sorCode"`
+  - Bad: `"jobCode”`
+- Property names should use full words, not abbreviations
+  - Good: `"propertyReference"`
+  - Bad: `"propertyRef”`
+
+## Data Formats
+
+- Dates and Times should be formatted as RFC3339:
+  - Don’t include milliseconds
+  - Always use UTC (represented with Z)
+  - Where just a date is required, still provide a full time object, with the time as 00:00:00
+  - Example time: `"2017-11-06T16:34:41Z"`
+  - Example date: `"2017-11-06T00:00:00Z"`
+- Durations should be represented as a number of minutes:
```

Comment:
[ISO8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) also defines a standard for durations, not sure if we want to consider using it, or stick with minutes for simplicity's sake.

Reply:
We discussed this and concluded that it looks like a standard which is comprehensive, but overengineered for our use case.

Reply:
For hackney purposes any duration can be represented in minutes. The standard includes years and months and across services I do not see any duration that will exceed a few days. (Even if it does current convention is to use From and To dates for those)